packages/@lwc/shared/src/language.ts

```diff
@@ -47,7 +66,7 @@ const {
     sort: ArraySort,
     splice: ArraySplice,
     unshift: ArrayUnshift,
-    forEach,
+    forEach, // Weird anomaly!
```

Comment:
We can rename this since it's just us. I don't know why we didn't call it `ArrayForEach`.